tests/lvmo/test_lvm_multi_clone.py

```diff
@@ -0,0 +1,322 @@
+    access_mode = constants.ACCESS_MODE_RWO
+    pvc_num = 5
+
+    @pytest.fixture()
+    def namespace(self, project_factory_class):
+        self.proj_obj = project_factory_class()
+        self.proj = self.proj_obj.namespace
+
+    @pytest.fixture()
+    def storageclass(self, volume_binding_mode, storageclass_factory_class):
+        if volume_binding_mode == constants.WFFC_VOLUMEBINDINGMODE:
+            sc_ocp_obj = OCP(kind="StorageClass", resource_name=constants.LVM_SC)
+            self.sc_obj = OCS(**sc_ocp_obj.data)
+        elif volume_binding_mode == constants.IMMEDIATE_VOLUMEBINDINGMODE:
+            sc_obj = templating.load_yaml(constants.CSI_LVM_STORAGECLASS_YAML)
+            sc_obj["metadata"]["name"] = create_unique_resource_name(
+                resource_description="immediate-test",
+                resource_type="storageclass",
+            )
+            sc_obj["volumeBindingMode"] = constants.IMMEDIATE_VOLUMEBINDINGMODE
+            self.sc_obj = storageclass_factory_class(custom_data=sc_obj)
+
+    @pytest.fixture()
+    def pvc(self, pvc_factory_class, volume_mode, volume_binding_mode):
+        self.status = constants.STATUS_PENDING
+        if volume_binding_mode == constants.IMMEDIATE_VOLUMEBINDINGMODE:
+            self.status = constants.STATUS_BOUND
+        futures = []
+        executor = concurrent.futures.ThreadPoolExecutor(max_workers=self.pvc_num)
+        for exec_num in range(0, self.pvc_num):
+            futures.append(
+                executor.submit(
+                    pvc_factory_class,
+                    project=self.proj_obj,
+                    interface=None,
+                    storageclass=self.sc_obj,
+                    size=self.pvc_size,
+                    status=self.status,
+                    access_mode=self.access_mode,
+                    volume_mode=volume_mode,
+                )
+            )
+        self.pvc_objs = []
+        for future in concurrent.futures.as_completed(futures):
+            self.pvc_objs.append(future.result())
+
+    @pytest.fixture()
+    def pod(self, pod_factory_class, volume_mode):
+        self.block = False
+        executor1 = concurrent.futures.ThreadPoolExecutor(max_workers=self.pvc_num)
+        if volume_mode == constants.VOLUME_MODE_BLOCK:
+            self.block = True
+        futures_pods = []
+        self.pods_objs = []
+        for pvc in self.pvc_objs:
+            futures_pods.append(
+                executor1.submit(pod_factory_class, pvc=pvc, raw_block_pv=self.block)
+            )
+        for future_pod in concurrent.futures.as_completed(futures_pods):
+            self.pods_objs.append(future_pod.result())
+
+    @pytest.fixture()
+    def run_io(self, volume_mode):
+        self.fs = "fs"
+        self.block = False
+        if volume_mode == constants.VOLUME_MODE_BLOCK:
+            self.fs = "block"
+            self.block = True
+        executor2 = concurrent.futures.ThreadPoolExecutor(max_workers=self.pvc_num)
```

Comment:
`executor1` is a `pod` method var and does not live outside of it so not sure we should start counting from 2 here

Reply:
done